scout/adapter/mongo/variant.py

```diff
@@ -557,9 +570,9 @@ def case_matching_causatives(self, case_obj, limit_genes=None):
                 # exclude causatives from the same case
                 continue
 
-            other_case = self.case(var_event["case"])
+            other_case = self.case(var_event["case"], CASE_CAUSATIVES_PROJECTION)
```

Comment:
This corresponds to the almost as good solution with projection from the gist given in #4100.